src/io/flutter/logging/FlutterLogEntry.java

```diff
@@ -27,6 +27,8 @@
   private final int level;
   @NotNull
   private final String message;
+  @Nullable
+  private String data;
```

Comment:
maybe make data a JsonElement?

Reply:
I started to do this actually (`Instance`) and then realized there may be other related changes.  I've added a TODO to revisit.

Thanks!